summariser/core/summariser.py

```diff
@@ -45,7 +45,7 @@ def __init__(self,
         logging.info(f"Num GPUs Available: {torch.cuda.device_count()}")
         logging.info(f"Model loaded")
 
-    def summarise(self, sentences: List[str]):
+    def summarise(self, sentences: List[str], max_length=None, min_length=None, num_beams=4):
```

Comment:
Remember to include types to help the developer
```suggestion
    def summarise(self, sentences: List[str], max_length: int = None, min_length: int = None, num_beams: int = 4):
```